internal/core/input.rs

```diff
@@ -145,18 +125,87 @@ impl Default for InputEventFilterResult {
     }
 }
 
-/// This module contains the constant character code used to represent the keys
+/// This module contains the constant character code used to represent the keys.
 #[allow(missing_docs, non_upper_case_globals)]
 pub mod key_codes {
     macro_rules! declare_consts_for_special_keys {
        ($($char:literal # $name:ident # $($_qt:ident)|* # $($_winit:ident)|* ;)*) => {
             $(pub const $name : char = $char;)*
+
+            #[allow(missing_docs)]
+            #[derive(Debug, Copy, Clone, PartialEq)]
+            #[non_exhaustive]
+            #[repr(C)]
+            /// The `Key` enum is used to map a specific key by name e.g. `Key::Control` to an
+            /// internal used unicode representation. The enum is convertible to [`std::char`] and [`crate::SharedString`].
+            /// Use this with [`crate::WindowEvent`](enum.WindowEvent.html) to supply key events to Slint's platform abstraction.
```

Comment:
```suggestion
            /// Use this with [`crate::WindowEvent`](`crate::api::WindowEvent`) to supply key events to Slint's platform abstraction.
```

(untested)  But I prefer always using this because it also works for the documentation of i_slint_core and will get warning if it's broken.

Reply:
Nice! I didn't know this worked, but it does and that's great.